db/migrations/20230227122704-ss58-prefix.js

```diff
@@ -40,6 +40,17 @@ class Logger {
     }
 }
 
+const decode = (address, ss58) => {
+    if (address.startsWith('0x')) {
+        console.log('Address starts with 0x', address);
+        return address;
+    } else {
+        return keyring.encodeAddress(
+            keyring.decodeAddress(address),
+            ss58);
+    }
```

Comment:
If we move out the return from the else block, the code will look flatter and more readable.